lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -34,7 +34,8 @@
 		</div>
 	</div>
 
-	<%= fields %>
+    <% // xss-lint: disable=underscore-not-escaped %>
```

Comment:
could that be indentation issue ?

Reply:
See the file and it shows correctly indented

![Screen Shot 2020-04-22 at 12 32 30 PM](https://user-images.githubusercontent.com/373677/79953519-694f0000-8495-11ea-9043-8aeddc73bf2e.png)